ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -232,7 +232,8 @@ private[apiserver] final class StoreBackedCommandExecutor(
           )
 
         case ResultNeedAuthority(holding @ _, requesting @ _, resume) =>
-          val granted = true // TODO #15882
+          // This file has been moved to canton. Nothing to do here.
```

Comment:
You problably still should put a refer to the issue, otherwise we cannot understand the comment.